gremlin-go/driver/graphBinary.go

```diff
@@ -1282,6 +1284,78 @@ func metricsReader(data *[]byte, i *int) (interface{}, error) {
 	return metrics, nil
 }
 
+type JanusgraphRelationIdentifier struct {
```

Comment:
This will be moved outside of the driver, but including here for an example of what an implementation could look like.